packages/edgestitch/lib/edgestitch/version.rb

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+module Edgestitch
+  VERSION = "1.0.0"
```

Comment:
are we ready to call this "good" and not "experimental" ?

Reply:
I think so… the test this went through was basically to be successful with the 150 nitro components + produce the same SQL as the previous implementation.